src/app/component/user/components/profile/profile-service/profile.service.ts

```diff
@@ -3,19 +3,38 @@ import { Observable } from 'rxjs';
 import { CardModel } from '@user-models/card.model';
 import { environment } from '@environment/environment';
 import { HttpClient } from '@angular/common/http';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+
 
 @Injectable({
   providedIn: 'root'
 })
 
 export class ProfileService {
 
-  constructor(private http: HttpClient) { }
+  constructor(private http: HttpClient,
+              private localStorageService: LocalStorageService) { }
 
   private backEnd = environment.backendLink;
+  public userId: number;
```

Comment:
Please, write your variables before constructor in such order: public first, then private

Reply:
fixed